src/website/views/components/stat/_examples.pug

```diff
@@ -594,128 +1094,138 @@ h2 Examples
               .chi-stat-submetric__title
                 | Upcoming
     .example-tabs.-pl--2
-      ul.chi-tabs#example-portal-base
+      ul.chi-tabs#example-portal-compact-base
         li.-disabled
-          a(href=`#webcomponent-portal-base`) Web Component
+          a(href=`#webcomponent-portal-compact-base`) Web Component
         li.-active
-          a(href=`#html-portal-base`) HTML Blueprint
-    .chi-tabs-panel#webcomponent-portal-base
-    .chi-tabs-panel.-active#html-portal-base
+          a(href=`#html-portal-compact-base`) HTML Blueprint
+    .chi-tabs-panel#webcomponent-portal-compact-base
+    .chi-tabs-panel.-active#html-portal-compact-base
       .chi-tab__description
         span.-text--grey
           | This HTML Blueprint requires JavaScript.
           | You may use your own solution, or use Chi's vanilla JavaScript solution,
           | <a href="../../../getting-started/installation">Chi.js</a>.
       :code(lang="html")
-        <div class="-overflow--auto">
-          <div class="chi-stat__wrapper">
-            <div class="chi-stat -compact">
-              <div class="chi-stat__content">
-                <div class="chi-stat-metric">
-                  <div class="chi-stat-metric__title">Utilization</div>
-                  <div class="chi-label__help">
-                    <button class="chi-button -icon -sm -flat -light" id="example__help-button" aria-label="Help" data-target="#example__help-popover" data-position="bottom"><i class="chi-icon icon-circle-info-outline" aria-hidden="true"></i></button>
-                  </div>
-                </div>
-                <div class="chi-stat-submetric">
-                  <div class="chi-stat-submetric__value">4</div>
-                  <div class="chi-stat-submetric__title">High</div>
-                </div>
-                <div class="chi-stat-submetric">
-                  <div class="chi-stat-submetric__value">16</div>
-                  <div class="chi-stat-submetric__title">Elevated</div>
+        <div class="chi-stat -compact">
+          <div class="chi-stat__item">
+            <div class="chi-stat__content">
+              <div class="chi-stat-metric">
+                <div class="chi-stat-metric__title">Utilization</div>
+                <div class="chi-label__help">
+                  <button class="chi-button -icon -xs -flat" id="example__portal-help-button" aria-label="Help" data-target="#example__portal-help-popover" data-position="bottom" aria-describedby="example__portal-help-popover">
```

Comment:
Looks like the <script> BUTTON IDs need to be updated with
id="example__`portal`-help-button"
id="example__`portal`-help-button2"
id="example__`portal`-help-button3"

Reply:
Maybe we can address this in another PR.